AppCenter/AppCenterTests/MSLogDBStorageTests.m

```diff
@@ -780,4 +786,13 @@ - (BOOL)logs:(NSArray<id<MSLog>> *)logs containLogWithSessionId:(NSString *)sess
   return NO;
 }
 
+- (void)validateQuery:(NSString *)query {
```

Comment:
I initially thought this method validates expected operations but the implementation only see the query is not malformed. The method name might lead wrong expectation, please add a doc for the method.